src/os/ui/draw/drawpicker.js

```diff
@@ -222,15 +224,16 @@ os.ui.draw.DrawPickerCtrl.prototype.initControlMenu = function() {
     });
   }
 
-  if (os.query.isCountryEnabled()) {
-    mi.addChild({
-      label: 'Country Border',
-      eventType: os.ui.menu.draw.EventType.COUNTRY_BORDER,
-      tooltip: 'Choose a country border',
-      icons: ['<i class="fa fa-fw fa-globe"></i> '],
-      handler: onDraw,
-      sort: 130
-    });
+  // add any configured OGC lookups (e.g. Country Borders)
+  var sort = 130;
+  for (const [, service, settings] of os.ogc.registry.getInstance().entries()) {
```

Comment:
Slight performance thing: for a loop like this, the to `getInstance().entries()` is happening on every iteration of the loop. In this case, it's not likely to matter, but for cases where that call has to generate, say, a 10000 element array every time, it's massively inefficient. Better to assign `const entries = os.ogc.registry.getInstance().entries()` and use the preassigned variable.